### PR TITLE
Update game

### PR DESCRIPTION
**Pull request by SR123456789-maker**
hi

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,51 +14,28 @@ _Collaborate and work together on GitHub._
...
+## Step 2: Assign yourself
 
-All great projects start with collaboration. Pull requests are the foundation of teamwork on GitHub — and pull request reviews give you the ability to work together and discuss changes specific to a pull request by commenting, requesting changes, or approving.
+_Great job opening that pull request! :wave:_
 
-- **Who is this for**: Developers, new GitHub users, users new to Git, students, managers, teams.
-- **What you'll learn**: When and how to request a review; how to provide a review of someone else's pull request.
-- **What you'll build**: We'll be reviewing a pull request for a simple game.
-- **Prerequisites**: We assume you are familiar with creating branches, commits, and pull requests—you can learn this in our [Introduction to GitHub](https://github.com/skills/introduction-to-github) course.
-- **How long**: This course takes less than 30 minutes to complete.
```
Comment (SR123456789-maker) [inline README.md:30]: hi

**Review by SR123456789-maker — COMMENTED** [review]
hi

**Comment by SR123456789-maker** [comment]
nice

File: README.md [hunk 2]
```diff
@@ -14,51 +14,28 @@ _Collaborate and work together on GitHub._
...
 
 <!--
-  <<< Author notes: Course start >>>
-  Include start button, a note about Actions minutes,
-  and tell the learner why they should take the course.
+  <<< Author notes: Step 2 >>>
+  Start this step by acknowledging the previous step.
+  Define terms and link to docs.github.com.
+  Historic note: this step combines the commend, approve, and needs changes steps from the previous version.
 -->
```
Comment (SR123456789-maker) [inline README.md:22]: sibfhf

Comment (SR123456789-maker) [reply README.md:22]: ni

File: README.md [hunk 3]
```diff
@@ -14,51 +14,28 @@ _Collaborate and work together on GitHub._
 </header>
 
 <!--
-  <<< Author notes: Course start >>>
-  Include start button, a note about Actions minutes,
-  and tell the learner why they should take the course.
+  <<< Author notes: Step 2 >>>
+  Start this step by acknowledging the previous step.
+  Define terms and link to docs.github.com.
```
Comment (saada1246-makers) [inline README.md:19]: ver nicce definitely

**Review by saada1246-makers — APPROVED** [review]
very nice

**Review by saada1246-makers — CHANGES_REQUESTED** [review]
very nice

**Comment by saada1246-makers** [comment]
@gsuf

File: README.md [hunk 4]
```diff
@@ -14,51 +14,30 @@ _Collaborate and work together on GitHub._
...
-## Welcome
+## Step 4: Suggest changes
 
-All great projects start with collaboration. Pull requests are the foundation of teamwork on GitHub — and pull request reviews give you the ability to work together and discuss changes specific to a pull request by commenting, requesting changes, or approving.
+_Nice work reviewing that pull request :sparkles:_
 
-- **Who is this for**: Developers, new GitHub users, users new to Git, students, managers, teams.
-- **What you'll learn**: When and how to request a review; how to provide a review of someone else's pull request.
-- **What you'll build**: We'll be reviewing a pull request for a simple game.
-- **Prerequisites**: We assume you are familiar with creating branches, commits, and pull requests—you can learn this in our [Introduction to GitHub](https://github.com/skills/introduction-to-github) course.
```
Comment (saada1246-makers) [inline README.md:29]: nice

**Review by saada1246-makers — COMMENTED** [review]
very

**Pull request merged**